test/mechanics/dofSubMatrixSolvers/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ endif()
 #add_unit_test(SpatialContainer)
```

Comment:
Whats up with these commented pieces of code? Some copy-paste remains?

Reply:
Was introduced by fe11df171e, I didn't touch it.